login.css

```diff
@@ -172,25 +189,23 @@ p{
   	top: 240px;
 }
 
-a{
-	color:black;
-	font-size:18px;
-	text-decoration:none;
-}
 
 #update{
 	color:black;
 	font-size:14px;
 	text-decoration:none;
+	font-family: 'Roboto', sans-serif;
+	font-weight:300;
 }
 
-.a{
```

Comment:
13/08/2021: This was use for the id update because the id update has a link but I released it was needed and it might cause confusion so I delete it.